docs/openapi/paths/admin.yaml

```diff
@@ -0,0 +1,125 @@
+get:
+  tags:
+    - admin
+  operationId: getUser
+  description: Gets one user from the database.
+  parameters:
+    - in: path
+      name: userId
+      schema:
+        type: integer
+      required: true
+      description: Numeric ID of the user to get
+  responses:
+    200:
+      description: One user by an id
+      content:
+        application/json:
+          schema:
+            type: object
+            $ref: '#/definitions/user'
+    500:
```

Comment:
We could remove these from the documentation.
No, we won't send them on purpose :-)

Most of the errors currently are hidden from the frontend or other clients of the API since we send a generic 500 error. This is by design to improve security while still having records of the errors in logs and db.

Reply:
I think we can leave out any errors that don't have some sort of surprising behavior (which hopefully we won't have, but leaving the possibility open), or that include meaningful information in the response for the client to deal with.